Mathlib/Topology/Algebra/ClopenNhdofOne.lean

```diff
@@ -0,0 +1,36 @@
+/-
+Copyright (c) 2024 Nailin Guan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nailin Guan, Yi Song, Xuchun Li
+-/
+import Mathlib.GroupTheory.Index
+import Mathlib.Topology.Algebra.ClosedSubgroup
+import Mathlib.Topology.Algebra.OpenSubgroup
+/-!
+## The Open Normal Subgroup in a Clopen Neighborhood of One
+This file define `OpenNormalSubgroupSubClopenNhdsOfOne` , which strengthen the existing result of
+`OpenSubgroupSubClopenNhdsOfOne` into open normal subgroup.
+
+This file is split out because the need of more imports.
+-/
```

Comment:
```suggestion

/-!
# Open normal subgroups in a clopen neighborhood of one

This file defines `OpenNormalSubgroupSubClopenNhdsOfOne`, which strengthens the result of
`OpenSubgroupSubClopenNhdsOfOne` into open *normal* subgroups.

This file is split out because the need of more imports.
-/
```

Reply:
Split out from where? Please mention the filename.